analysis_options.yaml

```diff
@@ -3,6 +3,9 @@
 include: package:flutter_lints/flutter.yaml
 
 analyzer:
+  exclude:
+    # Generated code from drift can contain unused local variables.
+    - lib/model/database.g.dart
```

Comment:
This should be reported as a bug in Drift — it's fine for the generated code to do this but then it should contain an ignore comment for it.

Reply:
Filed https://github.com/simolus3/drift/issues/3384 for that.